akka-stream/src/main/scala/akka/stream/impl/streamref/SinkRefImpl.scala

```diff
@@ -43,24 +50,30 @@ private[stream] final class SinkRefStageImpl[In] private[akka] (val initialPartn
       case OptionVal.None      => "<no-initial-ref>"
     }
 
-  override def createLogicAndMaterializedValue(inheritedAttributes: Attributes) = {
-    val promise = Promise[SourceRefImpl[In]]
+  override def createLogicAndMaterializedValue(inheritedAttributes: Attributes): (GraphStageLogic, SourceRef[In]) =
+    throw new IllegalStateException("Not supported")
```

Comment:
Can a user ever trigger this? If so, can we tell them what to do when they encounter this problem?

Reply:
Not in any reasonable scenario AFAICS